public/app/features/alerting/unified/utils/notification-policies.ts

```diff
@@ -160,9 +160,8 @@ function getInheritedProperties(
   childRoute: Route,
   propertiesParentInherited?: Partial<InhertitableProperties>
 ) {
-  const fullParentProperties = merge({}, parentRoute, propertiesParentInherited);
-
-  const inheritableProperties: InhertitableProperties = pick(fullParentProperties, INHERITABLE_KEYS);
+  const propsFromParent = pick(parentRoute, INHERITABLE_KEYS);
```

Comment:
The consequence of this change is that the `inheritableProperties` may contain fewer properties than in the original code. Only the properties in parentRoute that are in `INHERITABLE_KEYS` will be included in `propsFromParent` (in the previous code we also were including the whole parentRoute properties). Is this made intentionally?